examples/js/loaders/FBXLoader2.js

```diff
@@ -571,11 +571,10 @@
 
 							var faceVertexBuffer = [];
 							var polygonIndex = 0;
-							for ( var polygonVertexIndex = 0; polygonVertexIndex < indexBuffer.length; ++ polygonVertexIndex ) {
+							indexBuffer.forEach( function ( vertexIndex, polygonVertexIndex, indexBuffer ) {
```

Comment:
The following is just my speculation.
We need to ask @mrdoob for the real reason.

Yeah `forEach` has a good scope but `for` is faster than it in general.
Three.js (3DCG engine) often handles long array,
so only using `for` isn't bad I think.

    function init( array ) {
        for ( var i = 0; i < 0xffffff; i ++ ) {
		array[ i ] = 1;
        }
    }

    function run1( array ) {
        var sum = 0;
        for ( var i = 0, il = array.length; i < il; i ++ ) {
            sum += array[ i ];
        }
        return sum;
    }

    function run2( array ) {
        var sum = 0;
        array.forEach( function( value ) {
            sum += value;
        } );
        return sum;
    }

    function test(func, array) {
        var startTime = performance.now();
        func(array);
        var endTime = performance.now();
        console.log(endTime - startTime);
    }

    var array = [];
    init(array);
    test(run1, array); // warm up
    test(run1, array);
    test(run2, array); // warm up
    test(run2, array);

    // result
    VM73:48 18.36999999999989
    VM73:48 32.095000000001164
    VM73:48 504.89999999999964
    VM73:48 507.89999999999964

Reference: https://jsperf.com/for-vs-foreach/9